lib/utils.js

```diff
@@ -130,7 +130,7 @@ function patchEventTargetMethods(obj) {
   obj[keys.common.removeEventListener] = obj.removeEventListener;
   obj.removeEventListener = function (eventName, handler, useCapturing) {
     var eventType = eventName + (useCapturing ? '$capturing' : '$bubbling');
```

Comment:
From MDN: "Calling removeEventListener() with arguments that do not identify any currently registered EventListener on the EventTarget has no effect."

```
if (!handler) {
  return;
}
```

Is probably the best thing to do ?

Please also add test and check if `addEventListener` should be updated.